### PR TITLE
Describe intensions of some messages in onboarding doc

### PR DESCRIPTION
**Pull request by pirapira**
When I was reading the onboarding documents, I didn't understand what
each message does.  Now I'm trying to describe them after conversations.

### CODE REVIEW

File: docs/onboarding.rst [hunk 1]
```diff
@@ -36,11 +36,11 @@ The Happy Case
...
 
-Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.
+Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.  When ``Initiator`` receives the ``SecretRequest`` message, ``Initiator`` becomes sure that all intermediate channels have the required amount reserved for the pending transfer.
```
Comment (LefterisJP) [inline docs/onboarding.rst]: Perhaps the following sounds a bit better? Add a "the" and remove repetition.

```diff
-When ``Initiator`` receives the ``SecretRequest`` message, ``Initiator`` becomes sure that all intermediate channels have the required amount reserved for the pending transfer.
+When the ``Initiator`` receives the ``SecretRequest`` message, they become sure that all intermediate channels have the required amount reserved for the pending transfer.
```

File: docs/onboarding.rst [hunk 2]
```diff
@@ -36,11 +36,11 @@ The Happy Case
...
 
-When the ``Initiator`` gets the secret request message, they check to make sure that it's a valid one and that it corresponds to a locked transfer that they sent out. If all checks out they send a `Reveal Secret <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L710>`_ message back to the ``Target``.
+When the ``Initiator`` gets the secret request message, they check to make sure that it's a valid one and that it corresponds to a locked transfer that they sent out. If all checks out they send a `Reveal Secret <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L710>`_ message back to the ``Target``.  The ``RevealSecret`` message contains a secret that allows each `Mediator` or `Target` along the path to claim the locked amount in the pending transfer.
```
Comment (LefterisJP) [inline docs/onboarding.rst]: each ``Mediator`` or **the** ``Target``

Note the double `. You put single ones in there.

Comment (pirapira) [reply docs/onboarding.rst]: Shall I replace all double back quotes with single ones?

File: docs/onboarding.rst [hunk 3]
```diff
@@ -36,11 +36,11 @@ The Happy Case
...
 
-Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.
+Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.  When the ``Initiator`` receives the ``SecretRequest`` message, they sure that all intermediate channels have the required amount reserved for the pending transfer.
```
Comment (loredanacirstea) [inline docs/onboarding.rst]: `they sure that all intermediate` -> `they are sure`
maybe: 
```
he is sure that all intermediate channels have the required amount reserved for the pending transfer and that the mediated transfer has arrived to the ``Target``
```

File: docs/onboarding.rst (hunk 2 above)
Comment (pirapira) [reply docs/onboarding.rst]: Ah, I blame English.  "You put single ones" could mean "You did put single ones" or "Put single ones!".  I'll replace the single ones with doubles.

File: docs/onboarding.rst (hunk 3 above)
Comment (pirapira) [reply docs/onboarding.rst]: I used your sentence.  Just added a word "locked" before "transfer".

**Pull request merged**